include/storage/flash_map.h

```diff
@@ -46,22 +39,32 @@
 extern "C" {
 #endif
 
-#define SOC_FLASH_0_ID 0	/** device_id for SoC flash memory driver   */
-#define SPI_FLASH_0_ID 1	/** device_id for external SPI flash driver */
+/** Provided for compatibility with MCUboot */
+#define SOC_FLASH_0_ID 0
+/** Provided for compatibility with MCUboot */
+#define SPI_FLASH_0_ID 1
 
 /**
- * @brief Structure for store flash partition data
+ * @brief Flash partition
  *
- * It is used as the flash_map array entry or stand-alone user data. Structure
- * contains all data needed to operate on the flash partitions.
+ * This structure represents a fixed-size partition on a flash device.
+ * Each partition contains one or more flash sectors.
  */
 struct flash_area {
-	uint8_t fa_id; /** ID of flash area */
+	/** ID number */
+	uint8_t fa_id;
+	/** Provided for compatibility with MCUboot */
 	uint8_t fa_device_id;
 	uint16_t pad16;
```

Comment:
I doubt that `pad16` is required for anything.

Reply:
> I doubt that `pad16` is required for anything.

Are you asking for it to be removed? That's an API change not relevant to this PR.

Reply:
sure - this is somethin i can do by myself.